onnxruntime/core/providers/cuda/cuda_profiler.cc

```diff
@@ -93,7 +93,8 @@ void CUPTIAPI CudaProfiler::BufferCompleted(CUcontext, uint32_t, uint8_t* buffer
   free(buffer);
 }
 
-bool CudaProfiler::StartProfiling() {
+bool CudaProfiler::StartProfiling(TimePoint profiling_start_time) {
```

Comment:
>profiling_start_time [](http://example.com/codeflow?start=44&length=20)

nit: comment out the name might be fine enough.

Reply:
Done!